internal/registration/registration_test.go

```diff
@@ -13,7 +13,6 @@ import (
 	"fmt"
 	"io/ioutil"
 	"math/big"
-	"os"
```

Comment:
Why did you leave aliased `osUtil` instead of plain `os`?

Reply:
because `osUtil` alias was used and if someone put an alias there must a reason (I guess...) but yes, I agree with you that the alias is useless here, I can remove it if that good with you

Reply:
Please remove it if `os` doesn't clash with any local variable